pkg/format/format.go

```diff
@@ -23,18 +23,40 @@ const (
 var (
 	labelVar  string
 	fsTypeVar string
+	deviceVar string
 	drives    map[string]bool
 	driveKeys []string
 )
 
 func autoformat(label, fsType string) error {
 	var first string
 	for _, d := range driveKeys {
-		err := exec.Command("sfdisk", "-d", d).Run()
+		log.Printf("Considering auto format for device %s", d)
+
+		out, err := exec.Command("blkid", d).Output()
+		if err == nil {
+			log.Printf("Skipping autoformat of %s. blkid returned: %s", d, out)
+			continue
+		} else {
+			if exiterr, ok := err.(*exec.ExitError); ok {
+				if status, ok := exiterr.Sys().(syscall.WaitStatus); ok {
+					// blkid exitcode 2 (from the non-busybox version) signifies the block device has no detectable content signatures
+					if status.ExitStatus() == 2 {
+						log.Printf("blkid did not find any existing content on %s.", d)
```

Comment:
do not skip, as `blkid` exitcode 2 is returned when no content is detected (NOTE: only the `blkid` package from Alpine; the busybox `blkid` does not have this feature, so I install the alpine `blkid` package for this reason)

https://linux.die.net/man/8/blkid mentions this exitcode, should you like to see that man page